### ISSUE TITLE
please integrate latest eanthony github code to live site

### ISSUE THREAD

**Issue by ajdamico** [MonetDB/eanthony#28]
TL;DR -- please put the latest eanthony code on fedora20  thanks :)

---

our fix to brfss 1984[1] appears to have solved the 1984 problem and now it seems to be crashing at 1985 instead [2] so i think all of the brfss files were corrupted somehow?

i recently added a setting that will overwrite the cached files and still save them to the cache  [3]

maybe this temporary change [4] will fix the brfss issue?

[1] https://github.com/MonetDB/eanthony/issues/27
[2] http://monetdb.cwi.nl/testweb/web/eanthony/fedora20-3/1457251145/brfss.log
[3] https://github.com/ajdamico/asdfree/blob/master/Download%20Cache/download%20cache.R#L5-L6
[4] https://github.com/MonetDB/eanthony/commit/0964f0350e63ae2317c6183b20ad96418f4c28da

**Comment by hannes** [comment]
done

**Comment by ajdamico** [comment]
`pums` does not break at the same point on `rocks040` so let's try re-downloading the cache for that as well..  i've committed the change[1] please integrate when you have a minute? thanks homeslice

[1] https://github.com/MonetDB/eanthony/commit/69e52f98884b283d882809468ccd489dce1fa9b6

**Comment by ajdamico** [comment]
@hannesmuehleisen might have also solved the swmap_ces unix crash

https://github.com/MonetDB/eanthony/commit/38f5fde67f9e33be6bcf22fbb0eab59b91ab27ad
